lib/generators/active_admin/install/templates/active_admin.rb.erb

```diff
@@ -1,4 +1,12 @@
 ActiveAdmin.setup do |config|
+  # == Localize Date/Time Format
+  #
+  # Set the localize format to display dates and times.
+  # To understand how to localize your app with I18n, read more at
+  # https://github.com/svenfuchs/i18n/blob/master/lib%2Fi18n%2Fbackend%2Fbase.rb#L52
+  #
+  config.localize_format = :long
+
```

Comment:
Please move this section bevor `Setting a Favicon`, I want to keep a priority order